src/components/Stats.vue

```diff
@@ -1,53 +1,19 @@
-<template>
-  <ul class="stats">
-    <li>{{ downloads }} downloads</li>
-    <li>{{ stars }} stars</li>
-    <li>{{ contributors }} contributors</li>
-  </ul>

```

Comment:
Perhaps we can use `Number#toLocaleString()` on these values?
![image](https://user-images.githubusercontent.com/45381083/118910956-96612280-b8e2-11eb-9f9d-0c3864eba3ec.png)